hepcrawl/spiders/mit_spider.py

```diff
@@ -0,0 +1,221 @@
+# -*- coding: utf-8 -*-
+#
+# This file is part of hepcrawl.
+# Copyright (C) 2015, 2016 CERN.
+#
+# hepcrawl is a free software; you can redistribute it and/or modify it
+# under the terms of the Revised BSD License; see LICENSE file for
+# more details.
+
+"""Spider for MIT."""
+
+from __future__ import absolute_import, print_function
+
+import re
+
+from urlparse import urljoin
+
+import datetime
+import requests
+
+from scrapy.http import Request
+from scrapy.spiders import XMLFeedSpider
+
+from ..items import HEPRecord
+from ..loaders import HEPLoader
+from ..utils import get_temporary_file, split_fullname
+
+
+class MITSpider(XMLFeedSpider):
+
+    """MIT crawler
+    Scrapes theses metadata from DSpace@MIT (Dept. of Physics dissertations).
+    http://dspace.mit.edu/handle/1721.1/7608/browse
+
+    1. `get_list_file` makes post requests to get list of records as a html
+       file. Defaults are to take the current year and 100 records per file.
+    2. `parse` iterates through every record on the html page and yields
+       a request to scrape full metadata.
+    3. `build_item` builds the final HEPRecord.
+
+
+    Example usage:
```

Comment:
If you can also add the documentation statements I mentioned in the other PR as well we can merge this. Thanks!